VirtualReality/MRML/vtkVirtualRealityViewInteractorStyle.cxx

```diff
@@ -400,20 +304,12 @@ void vtkVirtualRealityViewInteractorStyle::OnMove3D(vtkEventData* edata)
   }
 
   //// Update rays
-  //if (this->HoverPick)
-  //{
-  //  this->UpdateRay(edd->GetDevice());
-  //}
+  this->UpdateRay(edd->GetDevice());
```

Comment:
Now that we derive from `vtkOpenVRInteractorStyle`, uncommenting this seems reasonable ?